src/test/resources/config/sidecar_multiple_instances.yaml

```diff
@@ -91,6 +91,10 @@ sidecar:
     cache:
       expire_after_access_millis: 7200000 # 2 hours
       maximum_size: 10000
+  sstable_snapshot:
+    snapshot_list_cache:
+      expire_after_access_millis: 350
```

Comment:
The expiry is really short.

Reply:
ah this is only a resource for tests, the actual value in `sidecar.yaml` is `7200000` or 2 hours